app/javascript/lib/study-overview/explore-single.js

```diff
@@ -59,7 +59,10 @@ async function renderSingleGenePlots(study, gene) {
 
   const geneList = window.SCP.formatTerms(gene)
   if (geneList.length > 1) {
+    // draw the dotPlot for comparison if this is a multi-gene collapsed by mean
     window.drawHeatmap()
+    // reattach the event handlers, since that call removes them
+    attachEventHandlers(study, gene)
```

Comment:
Thinking-out-load: I created `attachEventHandlers` as a way to encapsulate the old practice of attaching-and-reattaching event handlers.  But the underlying practice still seems a bit tedious and error-prone.

I suspect migrating these components to React will mitigate such footguns.